CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # CHANGELOG
 
+## v3.0.0-rc6 / 2018-05-24
```

Comment:
Would also mention `winston-transport@4.0.0` since this can break custom transports.